kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -8,30 +8,26 @@
     <kudos-api ref="kudosAPI" />
     <exo-drawer
       ref="activityKudosDrawer"
-      v-if="remainingKudos > 0"
       width="500px"
       hide-actions
-      @opened="drawer = true"
-      @closed="drawer = false"
       id="activityKudosDrawer"
-      allow-expand
       right
       disable-pull-to-refresh>
       <template slot="title">
-        <span class="text-capitalize-first-letter">
+        <span class="text-header-title">
           {{ $t('exoplatform.kudos.title.sendAKudos') }}
         </span>
       </template>
       <template slot="content">
```

Comment:
Please add a condition on the drawer content slot to not show it until the initialization is done.